combine-events/combine-events.test.ts

```diff
@@ -0,0 +1,796 @@
+/* eslint-disable prettier/prettier */
+import { createEvent, restore, Event } from 'effector';
+import { combineEvents } from './index';
+
+const argumentHistory = (fn) => fn.mock.calls.map(([argument]) => argument);
```

Comment:
You can import it from ../test-libary

Reply:
The version from `test-library.js` is different:
![image](https://user-images.githubusercontent.com/29141708/86052331-7d981600-ba5f-11ea-8ebb-bc3a5a70d6de.png)

Reply:
I have used the version from effector package. Why not you?

Reply:
What about to create issue to change our version to version from effector?

Reply:
Sure, but it is not a blocker for this PR

Reply:
#64